mobile/src/reducers/calendar/calendar.action.ts

```diff
@@ -13,9 +13,10 @@ export const loadCalendarEvents = (employeeId: string): LoadCalendarEvents => ({
 export interface LoadCalendarEventsFinished {
     type: 'LOAD-CALENDAR-EVENTS-FINISHED';
     calendarEvents: CalendarEvents;
+    employeeId: string;
 }
 
-export const loadCalendarEventsFinished = (calendarEvents: CalendarEvents): LoadCalendarEventsFinished => ({ type: 'LOAD-CALENDAR-EVENTS-FINISHED', calendarEvents });
+export const loadCalendarEventsFinished = (calendarEvents: CalendarEvents, employeeId?: string): LoadCalendarEventsFinished => ({ type: 'LOAD-CALENDAR-EVENTS-FINISHED', calendarEvents, employeeId });
```

Comment:
why employeeId optional? It'll probably break somewhere if this action doesn't have employeeId.

Reply:
ok, anyway I forgot to change the calendar side code and it still presents last requested employee events status instead of the current user - just realized that yesterday, so more work lasts here.